src/js/control-bar/control-bar.js

```diff
@@ -58,6 +59,7 @@ ControlBar.prototype.options_ = {
     'durationDisplay',
     'progressControl',
     'liveDisplay',
+    'seekToLive',
```

Comment:
As a general question, do we expect the live components to work independently of each other? i.e the seekToLive could be used without the liveDisplay

Reply:
`seekToLive` replaces `liveDisplay` but we wanted the player to be backwards compatible. Without the new option (`liveui`). `liveDisplay` will be shown since the player will only have the `vjs-live` class and `seekToLive` will be hidden since there is no `vjs-liveui` class. With the new option `seekToLive` will be shown since there will be a `vjs-liveui` class and `liveDisplay` will be hidden.

Reply:
That sounds like `seekToLive` should only be added to the children if `liveui` option is true. 

Also, seems I meant to ask if `liveDisplay` works with the liveTracker as well

Reply:
Thinking about it some more, we should just have it in by default and hidden by CSS. At some point, we should fix it so that just calling `component.show()` from a default-hidden component actually works.